src/monitor/node_metadata.c

```diff
@@ -453,10 +453,12 @@ ReportAutoFailoverNodeState(char *nodeName, int nodePort,
 		"UPDATE " AUTO_FAILOVER_NODE_TABLE
 		" SET reportedstate = $1, reporttime = now(), "
 		"reportedpgisrunning = $2, reportedrepstate = $3, "
-		"waldelta = CASE $4 WHEN -1 THEN waldelta ELSE $4 END, "
-		"walreporttime = CASE $4 WHEN -1 THEN walreporttime ELSE now() END, "
+		"reportedlsn = CASE $4 WHEN '0/0'::pg_lsn THEN reportedlsn ELSE $4 END, "
+		"walreporttime = CASE $4 WHEN '0/0'::pg_lsn THEN walreporttime ELSE now() END, "
 		"statechangetime = now() WHERE nodename = $5 AND nodeport = $6";
 
+	elog(INFO, "update query %s", updateQuery);
```

Comment:
Do we keep that in the merged PR? I don't think so?

Reply:
right, removed